daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -66,6 +68,8 @@ General guidance on upgrading can be found for [self hosted mode]({{<ref self-ho
 |                          |                 1.3.1 |                    1.4.1 |
 | 1.3.1                    |                   N/A |                    1.4.1 |
 | 1.4.0                    |                   N/A |                    1.4.1 |
+| 1.4.1                    |                 1.4.2 |                    1.4.3 |
+| 1.4.2                    |                   N/A |                    1.4.3 |
```

Comment:
you'll need to update lines 57-69 as well to extend the upgrade paths through to 1.4.3 - for example lines 58-61 should be extended to:

| 1.0.0 or 1.0.1           |                   N/A |                    1.1.2 |
|                                |                 1.1.2 |                    1.2.2 |
|                                |                 1.2.2 |                    1.3.1 |
|                                |                 1.3.1 |                    1.4.1 |
|                                |                 1.4.1 |                    1.4.2 |
|                                |                 1.4.2 |                    1.4.3 |